client/app/sessions/sessions.js

```diff
@@ -27,13 +27,11 @@ angular.module('dinnerDaddy.sessions', [])
     });
   };
 
-  // UNCOMMENT THIS WHEN IT WORKS: $scope.fetchSessions();
+  $scope.fetchSessions();
```

Comment:
move this function call into the 'then' for Auth.getUser, so it only gets called after we have the user token (no 403: Forbidden)